src/libs/actions/IOU.js

```diff
@@ -349,7 +362,7 @@ function requestMoney(report, amount, currency, payeeEmail, payeeAccountID, part
     }
 
     // STEP 3: Build optimistic transaction
-    const optimisticTransaction = TransactionUtils.buildOptimisticTransaction(amount, currency, iouReport.reportID, comment);
+    const optimisticTransaction = TransactionUtils.buildOptimisticTransaction(amount, currency, iouReport.reportID, comment, null, null, merchant);
```

Comment:
Just to confirm that this feature is still WIP and it's fine to pass `merchant` only here for now.
`buildOptimisticTransaction` is called in various places: when split money, send money, pay money request

Reply:
Yes this feature is still WIP - maybe we don't want to pass `merchant` yet, since it's currently not editable and we should handle this in a different issue where we make this field editable